### ISSUE TITLE
Upload fails, Connection is rejected

### ISSUE THREAD

**Issue by ramasek** [pfefferf/ui5-nwabap-deployer#18]
Hi Florian,

I was updating grunt-nwabap-ui5uploader from a rather old version (0.2.10).

With version 1.0.6 I get the following error:
"_The requested URL could not be retrieved"
"https://..../sap/bc/adt/filestore/ui5-bsp/objects/ZRM_UI5_SAMPLE/content?_"

The last version that is working for me is 1.0.3.
It seems to be the AdtClient change.

Best regards,
Ramón

**Comment by pfefferf** [comment]
1.0.6 works fine for me in my test (tested it just now).
Can you share your configuration?

**Comment by ramasek** [comment]
Maybe it is about the proxy configration?
			options: {
				conn: {
					server: 'https://<sapServer>:<port>',
					client: 200,
					useStrictSSL: false,
					proxy: 'http://<ip>:<port>'
				},
				auth: {
					user: sUser,
					pwd: sPwd
				},
				resources: {
					cwd: sFolder,
					src: aFiles
				}
			},
			upload_tmp: {
				options: {
					ui5: {
						language: 'DE',
						package: '$TMP',
						bspcontainer: 'ZRM_UI5_SAMPLE',
						bspcontainer_text: 'ZRM OpenUI5 Sample App'
					}
				}
			}

**Comment by pfefferf** [comment]
I have tested it now with a local http and https proxy implementation. Both ways work for me. Any further info which can help to reproduce the issue?

**Comment by ramasek** [comment]
Unfortunatly, i have no other idea.
I don't have much experience with Network infrastucture.

What I can say: It is the difference between 'axios' and 'request'.
I tried myself setting up both, requesting '/sap/bc/adt/discovery'.
Providing:
  - user/password 
  - client
  - language
  - proxy
  - strictSSL=false

It works for 'request'.
With 'axios' I receive 502 Bad Gateway.
Maybe it is something about strictSSL?

I also contacted my NET Admin, yet haven't got an answer.
